### DISCUSSION TITLE
package no longer works - it looks like Panasonic changed some things on the backend

### DISCUSSION THREAD

**Discussion by Shmow** [bisand/node-red-contrib-panasonic-comfort-cloud#20] [General]
As of last week the NodeRED-flow to control my AC no longer works and reports the following on every request:

```
{ 
responseMessage: "{"code":"4000"，"message":"Missing required header parameter or bad request for header"}",
httpCode: 400,
statusCode: 400,
statusMessage: "Bad Request"
}
```

I'm pretty sure I saw the Panasonic Comfort Cloud app getting updated too, but I can't find anything about it atm

**Comment by bisand** [comment]
Seems like Panasonic has changed something again. I will do some research to see if there is a quick fix.

**Comment by bisand** [comment]
This should be fixed by work done in https://github.com/bisand/node-red-contrib-panasonic-comfort-cloud/issues/19
